lib/Index/Index.cpp

```diff
@@ -1651,17 +1646,14 @@ bool IndexSwiftASTWalker::initIndexSymbol(ExtensionDecl *ExtD, ValueDecl *Extend
   if (getNameAndUSR(ExtendedD, ExtD, Info.name, Info.USR))
     return true;
 
-  std::tie(Info.line, Info.column, Info.offset) = getLineColAndOffset(Loc);
+  std::tie(Info.line, Info.column) = getLineCol(Loc);
   if (auto Group = ExtD->getGroupName())
     Info.group = Group.getValue();
   return false;
 }
 
 bool IndexSwiftASTWalker::initFuncDeclIndexSymbol(FuncDecl *D,
                                                   IndexSymbol &Info) {
-  if (initIndexSymbol(D, D->getLoc(/*SerializedOK*/false), /*IsRef=*/false, Info))
```

Comment:
What's the reason for this change? I find it confusing that `initFuncDeclIndexSymbol` does not imply `initIndexSymbol`.  There are several other `init<thing>IndexSymbol` methods and they all call `initIndexSymbol`.  I think we should keep them consistent, and it feels kind of unrelated to the rest of the PR.

Reply:
Personally I found this confusing 😅. I'd prefer `initIndexSymbol` be called explicitly and then the other "inits" be "addAdditional...". The 2 other `init*IndexSymbol` are both for refs, which I didn't change because one is used in a couple places (which I guess is a good reason to keep this).

Anyway, I don't feel particularly strongly about this so if you do I'm happy to change back.

FWIW it's "related" because I was going through all these calls when trying to find paths. It's a separate commit, happy to move to another PR (or, again, change back).

Reply:
Changed this back but left the rest 👍